app/utils/types.ts

```diff
@@ -786,7 +803,17 @@ export function instanceOfAccountTransactionWithSignature(
 export function instanceOfSimpleTransfer(
```

Comment:
I think the design is better if we keep the two transactions separate. So I would not expect `instanceOfSimpleTransfer` to match for one with a memo.